Java/core/src/main/java/com/amazon/randomcutforest/tree/AbstractRandomCutTree.java

```diff
@@ -276,24 +303,47 @@ private boolean deletePoint(NodeReference nodeReference, Point point, long seque
                 setParent(rootIndex, null);
             } else {
                 replaceNodeBySibling(grandParent, parent, nodeReference);
+                updateAncestorNodesAfterDelete(grandParent, point);
```

Comment:
Similarly, we have  both "recursively readjust" and "update ancestor nodes" which seem to be conveying pretty similar ideas. Can we standardize on something?

Reply:
removing the recursively from all names, stnadardizing on updateAncestorX